src/pages/experiments/[experimentId]/data-exploration/components/heatmap/HeatmapPlot.jsx

```diff
@@ -229,6 +230,14 @@ const HeatmapPlot = (props) => {
     // eslint-disable-next-line no-shadow
     const cartesian = (...a) => a.reduce((a, b) => a.flatMap((d) => b.map((e) => [d, e].flat())));
 
+    // Mapping between expressionValue with key to data
+    const dataDict = {
```

Comment:
I would call this something more specific to what it represents, like `expressionTypes` or something similar, just so that it is more explicit from the code what this dict is about.

Reply:
Changed the name to `dataSource` in c86dfad, and use it directly with `expressionValue` as key